addons/docs/src/frameworks/react/jsxDecorator.test.tsx

```diff
@@ -108,34 +108,32 @@ describe('renderJsx', () => {
   });
 
   it('forwardRef component', () => {
-    const MyExoticComponent = React.forwardRef(function MyExoticComponent(props: any, _ref: any) {
-      return <div>{props.children}</div>;
-    });
+    const MyExoticComponent = React.forwardRef((props: any, _ref: any) => (
+      <div>{props.children}</div>
+    ));
 
     expect(renderJsx(<MyExoticComponent>I'm forwardRef!</MyExoticComponent>, {}))
       .toMatchInlineSnapshot(`
-        <MyExoticComponent>
-          I'm forwardRef!
-        </MyExoticComponent>
-      `);
+      <[object Object]>
```

Comment:
something is stripping off displayName??

Reply:
Yeah, but we don't know what..